chcemvediet/apps/accounts/tests/test_backend.py

```diff
@@ -0,0 +1,107 @@
+from django.contrib.auth.decorators import user_passes_test
+from django.conf.urls import patterns, url, RegexURLPattern
+from django.contrib.auth.models import User
+from django.http import HttpResponse
+from django.test import TestCase
+from django.test.utils import override_settings
+
+
+class AdminLoginAsBackendTest(TestCase):
+
+    def public_view(request):
+        if isinstance(request.user, User):
+            pass  # force request.user to evaluate
+        return HttpResponse()
+
+    @user_passes_test(lambda u: u.is_staff)
+    def admin_view(request):
+        if isinstance(request.user, User):
+            pass
+        return HttpResponse()
+
+    @user_passes_test(lambda u: u.is_staff)
+    def set_admin_login_as_attribute_admin_view(request, id):
```

Comment:
Pls parametrer volaj rovnako ako v admin endpoint `obj_pk`. Kedze je to to iste.

Reply:
:ok: